### PR TITLE
Add new regions for SSPE

### PR DESCRIPTION
**Pull request by suehighmoor**
Include new regions in references to availability of SSPE

### CODE REVIEW

File: articles/portal/ptl-ref-portal-api.md [hunk 1]
```diff
@@ -1036,7 +1036,7 @@ Parameter name | Description | Type | Mandatory (Default)
 ---------------|-------------|------|--------------------
 type | The object type; must be Vorg | String | Y
 name | The name of the vOrg | String | Y
-zone id | The zone in which to create the vOrg</br>Valid values:</br>- B (for Region 5, Assured)</br>- D (for Region 5, Elevated)</br>- F (for Region 6, Assured)</br>- 12 (for Region 6, Elevated) | String | Y
+zone id | The zone in which to create the vOrg</br>Valid values:</br>- B (for region 5, Assured)</br>- D (for region 5, Elevated)</br>- F (for region 6, Assured)</br>- 12 (for region 6, Elevated)<br>-2D (region 13, Assured)<br>-2E (region 14, Assured) | String | Y
```
Comment (bensaunders) [inline articles/portal/ptl-ref-portal-api.md:1039]: A minor point, but there's a space after the hyphen in the existing bullet points, which is missing in the new ones.

Comment (suehighmoor) [reply articles/portal/ptl-ref-portal-api.md]: Thanks! All fixed.

**Comment by myoung-ukcloud** [comment]
I'm ready to approve. The only comment I have is that there are a lot of other places throughout the documentation repository where regions 5 & 6 are specifically mentioned. Have all of those areas been checked to see whether they should be adapted to mention regions 13 & 14 too? 

e.g.

I found line 115 in https://github.com/UKCloud/documentation/blob/master/articles/dedicated-compute/dc-sco.md where it says 'Dedicated Compute v2 is available only in UKCloud Regions 5 and 6.'. I'm not in a position to judge whether that should be changed. I thought it worth asking whether it had been considered and didn't need updating?

Also https://github.com/UKCloud/documentation/blob/master/articles/enablement/enbl-how-zerto-migrate-between-regions.md where it says 'the target environment for Journaling Protection for failover is available only in regions 5 and 6.'

There are many other references too.

**Comment by suehighmoor** [comment]
@myoung-ukcloud, thanks for the feedback. Let's keep this pull request for the SSPE stuff and I'll create another task for updating other services.

**Review by myoung-ukcloud — APPROVED** [review]
LGTM

**Review by vrush20 — APPROVED** [review]
LGTM

**Pull request merged**